src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Figure extends FigureColor {
+    double calculatedArea();
```

Comment:
There is a typo in the method name 'calculatedArea'. It should be 'calculateArea' to follow standard naming conventions and ensure consistency with the implementing classes.